build.gradle

```diff
@@ -92,8 +92,22 @@ subprojects {
     tasks.check.dependsOn(tasks.jacocoTestReport)
     tasks.check.dependsOn(javadoc)
 
-    if (System.env.CIRCLE_TEST_REPORTS) {
-        test.reports.junitXml.destination = new File(System.env.CIRCLE_TEST_REPORTS, it.getName())
+    if (System.env.CIRCLECI) {
+        test.reports.junitXml.destination = new File(System.env.TESTS_DIR, it.getName())
+
+        task copyCiArtifactsToCircle {
```

Comment:
Haha this change was in response to @bavardage asking that the logic be moved out of `circle.yml` and into Gradle :)

See https://github.com/palantir/encrypted-config-value/pull/69/files/e9b315f22c5f18dcb5e85d01ac89df0597a3735f for what this looks like in the `circle.yml` file -- we can definitely do it, but down-side is that the project mapping is fragile